src/outlet/index.ts

```diff
@@ -121,25 +120,60 @@ export class Outlet {
 						const attestationBlob = this.getDataFromQuery('attestation')
 						const attestationSecret = '0x' + this.getDataFromQuery('requestSecret')
```

Comment:
@nicktaras , could you add check if query params exist , non-empty and contains data in correct format?

Reply:
Let's move this to a seperate Jira task, where this is a change related to previous work. Thanks for picking this up to look into.

Reply:
https://smarttokenlabs.atlassian.net/browse/TIKAT-294